app/components/chart-donut.js

```diff
@@ -43,6 +43,14 @@ export default Component.extend({
       return;
     }
 
+    if (!isIcon) {
+      svg.classed("svg-container", true)
+      .attr("preserveAspectRatio", "xMinYMin meet")
+      .attr("viewBox","0 0 " + width + " " + height)
+      //class to make it responsive
+      .classed("svg-content-responsive", true);
```

Comment:
Do we need both of these classes? Shouldn't we just make all SVGs responsive? Or put another way - is there an `.svg-container` that isn't also going to be responsive?